jest.config.js

```diff
@@ -44,6 +44,6 @@ module.exports = {
     '^.+\\.(txt)$': require.resolve('./node_modules/react-native/jest/assetFileTransformer.js'),
   },
   transformIgnorePatterns: [
-    'node_modules/(?!(@celo/)?@?react-native|@react-navigation|@react-native-community|uuid|statsig-js|@react-native-firebase|react-navigation|redux-persist|date-fns|victory-*|@walletconnect/react-native-compat)',
+    'node_modules/(?!(@celo/)?@?react-native|@react-navigation|@react-native-community|uuid|statsig-js|@react-native-firebase|react-navigation|redux-persist|date-fns|victory-*|@walletconnect/react-native-compat|react-redux)',
```

Comment:
This was needed to avoid the following error, when running jest:

```
/Users/jean/src/github.com/valora-inc/wallet/node_modules/react-redux/dist/react-redux.legacy-esm.js:34
    import * as React2 from "react";
    ^^^^^^

    SyntaxError: Cannot use import statement outside a module
```